src/graphql/schema/Trip/typeDefs.gql

```diff
@@ -18,6 +18,15 @@ extend type Mutation {
   unlockDestination(tripID: ID!): Trip!
   unlockTimeFrame(tripID: ID!): Trip!
   unlockBudget(tripID: ID!): Trip!
+  addChatMessage(tripID: ID!, message: String!): Trip!
```

Comment:
That one can be removed as it is replaced by addGeneralMessage and it's buddies addTimeFrameMessage, addBudgetMessage and addDestinationMessage, right?

- [ ] remove addChatMessage from typeDefs